src/util.js

```diff
@@ -457,11 +457,16 @@ const util = {
     return os.cpus().length;
   },
 
+  /**
+   * Test email format based on W3C HTML5 specification.
+   * This check is not exaustive, and includes a willful violation of RFC5322
+   * (see https://html.spec.whatwg.org/multipage/input.html#email-state-(type=email))
```

Comment:
Might make sense to give some justification for using it here, e.g.
```suggestion
   * This check is not exaustive, and does not match RFC 5322 exactly
   * (see https://html.spec.whatwg.org/multipage/input.html#email-state-(type=email)),
   * but is commonly used for email address validation.
```